pom.xml

```diff
@@ -408,7 +408,7 @@
                 https://github.com/jboss/xerces/pull/14
                 https://issues.apache.org/jira/browse/XERCESJ-1276
             -->
-            <version>2.11.0.SP6-RB</version>
+            <version>2.11.0.SP6-RB-SNAPSHOT</version>
```

Comment:
This dependency is unfortunately only on our nexus. Not sure if adding *-SNAPSHOT* helps. But it might be possible for you to remove the dependency (slightly impacts performance of xml xsd validation). Or we can provide the jar for you. Anyways, this is not relevant to the PR.